draft-ietf-quic-invariants.md

```diff
@@ -143,11 +143,13 @@ version-specific semantics are marked with an X.
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                         Version (32)                          |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|DCIL(4)|SCIL(4)|
+|    DCIL (8)   |
```

Comment:
Maybe not for this PR, but something to consider...

"DCIL" was chosen to fit in the small space available in the diagram.  The new field is a little wider and could fit `DCID Len (8)` comfortably.